components/layout/DocsLayout.js

```diff
@@ -177,6 +178,20 @@ export default function DocsLayout({ post, navItems = {}, children }) {
                   {generateEditLink(post)}
                 </p>
               </div>
+              { (post.releaseNoteLink !== undefined)? 
+                // show only when it is related to specification (/docs/reference/specification) AND is not a pre-release 
+                // for example, if the post's title is "3.0.0 (Pre-release)", which will not have RN, so do not render this section. 
+                <div className="w-full mt-5 p-3 text-center rounded-lg border border-gray-600 bg-secondary-100">
+                  <span className="text-sm font-sans antialiased text-gray-800"> 
+                    {`What is new in v${post.title}? Have a look at `} 
+                  </span> 
+                  <Link href={post.releaseNoteLink}> 
+                    <a target="_blank" rel="noopener noreferrer" className={`font-body text-sm leading-6 font-medium text-secondary-500 hover:text-secondary-600 focus:outline-none focus:text-gray-900 transition ease-in-out duration-150`}> Release Note </a> 
+                  </Link> 
+                </div> 
+                : 
+                <></> 
+                }
```

Comment:
```suggestion
              { post?.releaseNoteLink && 
                // show only when it is related to specification (/docs/reference/specification) AND is not a pre-release 
                // for example, if the post's title is "3.0.0 (Pre-release)", which will not have RN, so do not render this section. 
                <div className="w-full mt-5 p-3 text-center rounded-lg border border-gray-600 bg-secondary-100">
                  <span className="text-sm font-sans antialiased text-gray-800"> 
                    {`What is new in v${post.title}? Have a look at `} 
                  </span> 
                  <Link href={post.releaseNoteLink}> 
                    <a target="_blank" rel="noopener noreferrer" className={`font-body text-sm leading-6 font-medium text-secondary-500 hover:text-secondary-600 focus:outline-none focus:text-gray-900 transition ease-in-out duration-150`}> Release Note </a> 
                  </Link> 
                </div> 
                }
```

I prefer this style of writing this component because we don't have any `else` condition in this case scenario.

Reply:
@akshatnema  Done :smile:. Please have a look at it